core/bin/zksync_api/src/bin/providers/dev_price_provider.rs

```diff
@@ -140,22 +179,33 @@ pub fn create_price_service(sloppy_mode: bool) -> actix_web::Scope {
         .chain(testnet_tokens.into_iter())
         .collect();
     if sloppy_mode {
-        web::scope(API_PATH)
+        web::scope("")
             .app_data(web::Data::new(data))
             .route(
-                "/coins/list",
+                "/cryptocurrency/quotes/latest",
+                web::get().to(make_sloppy!(handle_coinmarketcap_token_price_query)),
+            )
+            .route(
+                format!("{}/coins/list", API_PATH).as_str(),
                 web::get().to(make_sloppy!(handle_coingecko_token_list)),
             )
             .route(
-                "/coins/{coin_id}/market_chart",
+                format!("{}/coins/{{coin_id}}/market_chart", API_PATH).as_str(),
                 web::get().to(make_sloppy!(handle_coingecko_token_price_query)),
             )
     } else {
-        web::scope(API_PATH)
+        web::scope("")
```

Comment:
But if we remove this, then the endpoint will be changed, isnt it?

We do not longer will have the `/api/v3`.

Reply:
The `/api/v3` part has been moved to the specific handlers that require it, see lines 204 and 208.

Reply:
I see, thank you :)